codalab/server/json_api_plugin.py

```diff
@@ -0,0 +1,15 @@
+"""
+Utility functions for building and processing JSON API responses and requests.
+"""
+from bottle import response
+
+
+class JsonApiPlugin(object):
+    api = 2
+
+    def apply(self, callback, route):
+        def wrapper(*args, **kwargs):
+            response.content_type = 'application/vnd.api+json'
```

Comment:
This isn't necessary since by default if you return a dict the response content type is set to application/json by the JsonPlugin in bottle. In fact, this setting is override by that plugin.